### PR TITLE
Add Flying Shuttle tests

### PR DESCRIPTION
**Pull request by Timer**

### CODE REVIEW

File: test/integration/flying-shuttle/test/index.test.js [hunk 1]
```diff
@@ -0,0 +1,203 @@
...
+
+describe('Flying Shuttle', () => {
+  beforeAll(async () => {
+    {
+      await fs.remove(path.join(appDir, '.next-first'))
+      await fs.mkdirp(path.join(appDir, '.next-first'))
+
+      const { stdout } = await runNextCommand(['build', appDir], {
+        stdout: true,
+        stderr: true
```
Comment (ijjk) [inline test/integration/flying-shuttle/test/index.test.js]: Doesn't look like you're using `stdout` so could drop it here

File: test/integration/flying-shuttle/test/index.test.js [hunk 2]
```diff
@@ -0,0 +1,203 @@
...
+jasmine.DEFAULT_TIMEOUT_INTERVAL = 1000 * 60 * 5
+
+describe('Flying Shuttle', () => {
+  beforeAll(async () => {
+    {
+      await fs.remove(path.join(appDir, '.next-first'))
+      await fs.mkdirp(path.join(appDir, '.next-first'))
+
+      const { stdout } = await runNextCommand(['build', appDir], {
+        stdout: true,
```
Comment (Timer) [inline test/integration/flying-shuttle/test/index.test.js]:
```suggestion
        stdout: true
```

File: test/integration/flying-shuttle/test/index.test.js [hunk 3]
```diff
@@ -0,0 +1,202 @@
...
+        path.join(appDir, 'pages', 'about', 'index.js.hide')
+      )
+      await fs.rename(
+        path.join(appDir, 'pages', 'about', 'index.js2'),
+        path.join(appDir, 'pages', 'about', 'index.js')
+      )
+
+      const { stdout } = await runNextCommand(['build', appDir], {
+        stdout: true,
+        stderr: true
```
Comment (ijjk) [inline test/integration/flying-shuttle/test/index.test.js]:
```suggestion
```

File: test/integration/flying-shuttle/test/index.test.js [hunk 4]
```diff
@@ -0,0 +1,202 @@
...
+        path.join(appDir, 'pages', 'about', 'index.js'),
+        path.join(appDir, 'pages', 'about', 'index.js.hide')
+      )
+      await fs.rename(
+        path.join(appDir, 'pages', 'about', 'index.js2'),
+        path.join(appDir, 'pages', 'about', 'index.js')
+      )
+
+      const { stdout } = await runNextCommand(['build', appDir], {
+        stdout: true,
```
Comment (Timer) [inline test/integration/flying-shuttle/test/index.test.js]:
```suggestion
        stdout: true
```

**Review by ijjk — APPROVED** [review]

**Pull request merged**